web/lib/supabase/users.ts

```diff
@@ -47,3 +47,57 @@ export async function searchUsers(prompt: string, limit: number) {
     limit
   )
 }
+
+export async function searchUsersExcludingArray(
```

Comment:
Hmm, might be better to have `searchUsers` call this function with an empty array to reduce code duplication